workos/utils/connection_types.py

```diff
@@ -4,5 +4,13 @@
 class ConnectionType(Enum):
     ADFSSAML = "ADFSSAML"
     AzureSAML = "AzureSAML"
+    GenericOIDC = "GenericOIDC"
+    GenericSAML = "GenericSAML"
     GoogleOAuth = "GoogleOAuth"
+    GoogleSAML = "GoogleSAML"
+    MagicLink = "MagicLink"
     OktaSAML = "OktaSAML"
+    OneLoginSAML = "OneLoginSAML"
+    PingFederateSAML = "PingFederateSAML"
+    PingOneSAML = "PingOneSAML"
+    VMwareSAML = "VMwareSAML"
```

Comment:
no salesforce?

Reply:
Not yet! Salesforce is still behind a feature flag and not available in GA.